config/config.go

```diff
@@ -25,3 +31,31 @@ const (
 	// if the cluster is OSD (managed) or not
 	ClusterDeploymentManagedLabel string = "api.openshift.com/managed"
 )
+
+var isFedramp = false
+
+// SetIsFedramp gets the value of fedramp
+func SetIsFedramp() error {
+	// Get fedramp var
+	fedramp, ok := os.LookupEnv("FEDRAMP")
+	if !ok {
+		// If lookup fails, default to false
+		fedramp = "false"
+	}
+
+	fedrampBool, err := strconv.ParseBool(fedramp)
+	if err != nil {
+		return fmt.Errorf("Invalid value for FedRAMP environment variable. %w", err)
+	}
+
+	isFedramp = fedrampBool
+	if isFedramp {
+		fmt.Println("Running in fedramp environment.")
```

Comment:
This function is responsible for setting the variable up not sure you want to just print to console here.